app/src/main/res/values/strings.xml

```diff
@@ -158,16 +158,16 @@
     <string name="app_loyalty_card_keychain">Loyalty Card Keychain</string>
 
     <string name="privacy_policy">Privacy Policy</string>
-    <string name="privacy_policy_popup_text">Privacy policy notice (required by some app stores):\n\nWe collect NO DATA AT ALL, which anyone can confirm since our app is libre software.</string>
+    <string name="privacy_policy_popup_text">Privacy policy notice (required by some app stores):\n\nNO DATA IS COLLECTED AT ALL, which anyone can confirm since our app is libre software.</string>
     <string name="accept">Accept</string>
     <string name="importCatima">Import from Catima</string>
-    <string name="importCatimaMessage">Please select your Catima export file. It is most likely named Catima.csv.\n\nA Catima export file can be created by going to the Import/Export menu and pressing "Export".</string>
+    <string name="importCatimaMessage">Find a file likely named <i>Catima.csv</i> to import.\nOr create it from the Import/Export menu of another Catima app and pressing "Export" first.</string>
```

Comment:
```suggestion
    <string name="importCatimaMessage">Find a file likely named <i>Catima.csv</i> to import.\nOr create it from the Import/Export menu of another Catima app by pressing "Export" there first.</string>
```

Reply:
Shouldn't we rather phrase this like:

"Find the Catima.csv file to import from. If you don't have such a file, it can be created in the Catima app by going to the Import/Export menu and pressing Export there."

So we make it clear that you need to create this file to be able to import it?